cmd/routed-eni-cni-plugin/cni.go

```diff
@@ -318,19 +319,16 @@ func del(args *skel.CmdArgs, cniTypes typeswrapper.CNITYPES, grpcClient grpcwrap
 }
 
 func main() {
-	logger.SetupLogger(logger.GetLogFileLocation(defaultLogFilePath))
-
+	log := logger.Default()
 	log.Infof("Starting CNI Plugin %s ...", version)
```

Comment:
I would actually remove the above log line. It's not correct. :) The CNI plugin is not "started". Kubelet simply executes the CNI plugin, passing an ADD or DEL request to it. So, all the above log line does is clutter log output for kubelet, which will have logs that looks like this:

```
Starting CNI Plugin 1
Received CNI add request: ContainerID(blah) Netns(blah) IfName(eth0) Args(blah) Path(blah) argsStdinData(blah)
Starting CNI Plugin 1
Received CNI add request: ContainerID(blah) Netns(blah) IfName(eth0) Args(blah) Path(blah) argsStdinData(blah)
Starting CNI Plugin 1
Received CNI add request: ContainerID(blah) Netns(blah) IfName(eth0) Args(blah) Path(blah) argsStdinData(blah)
```

Reply:
I agree, I will remove it.